packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -17,13 +17,39 @@ attributes. Resources that expose metrics will have functions that look
 like `metricXxx()` which will return a Metric object, initialized with defaults
 that make sense.
 
+#### Exposed Metrics
+
 For example, `lambda.Function` objects have the `fn.metricErrors()` method, which
 represents the amount of errors reported by that Lambda function:
 
 ```ts
 const errors = fn.metricErrors();
 ```
 
+#### Constructed Metrics
+
+Constructed metrics could be a regular metric or a math expression.
```

Comment:
Who is to say constructs cannot expose predefined math expressions? You're excluding that possibility by using this wording. Point is you shouldn't need to know.